src/identity/interaction/util/BaseEmailSender.ts

```diff
@@ -0,0 +1,41 @@
+import { createTransport } from 'nodemailer';
+import type Mail from 'nodemailer/lib/mailer';
+import { EmailSender } from './EmailSender';
+import type { EmailArgs } from './EmailSender';
+
+export interface EmailSenderArgs {
+  emailConfig: {
+    name: string;
```

Comment:
What name is this?

Reply:
Name of the person who sends the mail.

Reply:
Surely that must be… `senderName` below?

Reply:
Ah yea, good point, answered too fast. :D 

Had to look in the nodemailer documentation, apparently it is
>  optional hostname of the client, used for identifying to the server, defaults to hostname of the machine

Reply:
Actually, I'm not even sure it's that since from how I read it that would need to be in a subfield. We can probably drop this one.

Reply:
Up to you